### PR TITLE
support for TABLE prefix operator

### PR DESCRIPTION
**Pull request by kanha-gupta**
Ref #24200 
@strongduanmu Please review :)
support for sqlCaseID - select_from_with_table
SelectStatementConverter cannot process queries with null Projections therefore I have optimised the code to process it for this task.
Any changes required ?

Screenshot (332)

### CODE REVIEW

**Comment by strongduanmu** [comment]
Can you give an example to show this case?

File: parser/sql/dialect/mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/MySQLStatementVisitor.java [hunk 1]
```diff
@@ -761,7 +761,12 @@ public ASTNode visitQuerySpecification(final QuerySpecificationContext ctx) {
     @Override
     public ASTNode visitTableStatement(final TableStatementContext ctx) {
         MySQLSelectStatement result = new MySQLSelectStatement();
-        result.setTable((SimpleTableSegment) visit(ctx.tableName()));
+        if (null != ctx.TABLE()) {
```
Comment (strongduanmu) [inline parser/sql/dialect/mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/MySQLStatementVisitor.java:764]: Can you add sql parser test case for this change?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Comment by kanha-gupta** [comment]
> Can you give an example to show this case?

Done :)

**Comment by kanha-gupta** [comment]
@strongduanmu please review :)

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**